src/community/label_propagation.c

```diff
@@ -251,6 +251,7 @@ igraph_error_t igraph_community_label_propagation(const igraph_t *graph,
         IGRAPH_FINALLY(igraph_vector_int_destroy, &node_order);
     }
 
+    control_iteration = 1;
```

Comment:
Please add a comment here explaining what `control_iteration` means when it's 1 and what it means when it's zero. (I understand it by reading the code now, but I'm not sure I'd still remember it if I look at the same piece of code two months later).